ui/pages/swaps/view-quote/view-quote.js

```diff
@@ -99,6 +103,10 @@ import CountdownTimer from '../countdown-timer';
 import SwapsFooter from '../swaps-footer';
 import ViewQuotePriceDifference from './view-quote-price-difference';
 
+// eslint-disable-next-line prefer-destructuring
+const EIP_1559_V2_ENABLED =
```

Comment:
It would be good to have a selector for EIP 1559 v2, like we have for EIP 1559 inside `ducks/metamask/metamask.js`:
```
export function isEIP1559Network(state) {
  return state.metamask.networkDetails?.EIPS[1559] === true;
}
```

Reply:
Yep but actually `EIP 1559 v2` is `isEIP1559Network(state) + feature_flag_enabled` so it is not purely a state thing.

Reply:
All that logic can be abstracted into a selector, so we don't need to duplicate this every time we need it:
```
const EIP_1559_V2_ENABLED = process.env.EIP_1559_V2 === true || process.env.EIP_1559_V2 === 'true';
const supportsEIP1559V2 = EIP_1559_V2_ENABLED && networkAndAccountSupports1559;
```

Reply:
I would actually avoid adding logic to selector, a separate hook or utility function might be actually a better place for it.